### PR TITLE
docs: Fix url in policy-storage

### PR DESCRIPTION
**Pull request by ErikQQY**
Fix url in [Policy Storage](https://casbin.org/docs/zh-CN/policy-storage)

Signed-off-by: ErikQQY <2283984853@qq.com>

### CODE REVIEW

File: docs/PolicyStorage.md [hunk 1]
```diff
@@ -3,4 +3,5 @@ id: policy-storage
 title: Policy Storage
 ---
 
-In Casbin, the policy storage is implemented as an adapter. Please go to: [/docs/en/adapters](/docs/en/adapters)
+In Casbin, the policy storage is implemented as an adapter. Please go to [Adapters](https://casbin.org/docs/en/adapters) for more details.
```
Comment (hsluoyz) [inline docs/PolicyStorage.md:6]: use relative path.

Actually I think it's a Chinese translation issue.

Comment (ErikQQY) [reply docs/PolicyStorage.md:6]: The relative path /docs/en/adapters doesn't exist, ./adapters would work

[](https://imgtu.com/i/ghGu6g)

While other files use the absolute path, so I fix it to the absolute path.

Comment (hsluoyz) [reply docs/PolicyStorage.md:6]: @ErikQQY it works: https://casbin.org/docs/en/policy-storage

Comment (ErikQQY) [reply docs/PolicyStorage.md:6]: The Chinese translation is here:

[](https://imgtu.com/i/ghJ6Vs)

I don't why it is broken in Chinese docs.

Comment (ErikQQY) [reply docs/PolicyStorage.md:6]: @hsluoyz The link in Chinese translation works:

https://casbin.org/docs/zh-CN/policy-storage

Comment (hsluoyz) [reply docs/PolicyStorage.md:6]: Great! I will close here.

**Pull request closed without merging**